slides/courses/4/number.tex

```diff
@@ -215,7 +216,8 @@ \subsection{MUL}
 
 \begin{frame}
     \frametitle{Booth's Algorithm}
- Booth's algorithm is a multiplication algorithm that multiplies two signed binary numbers in two-component notation. The algorithm reduces the number of additions required for the multiplication of two numbers.
+ Booth's algorithm is a multiplication algorithm that multiplies two signed binary numbers in two-component notation.
```

Comment:
multiplication algorithm that multiplies two signed binary numbers in two-component notation.

Shouldn't it be two's-complement? Instead